diff --git a/kernel/feature/sucompat.c b/kernel/feature/sucompat.c
index b5d3313d..aac8e143 100644
--- a/kernel/feature/sucompat.c
+++ b/kernel/feature/sucompat.c
@@ -7,6 +7,7 @@
 #include <asm/pgtable.h>
 #endif
 #include <linux/uaccess.h>
+#include <linux/printk.h>
 #include <asm/current.h>
 #include <linux/cred.h>
 #include <linux/fs.h>
@@ -396,1 +397,1 @@ static inline void ksu_handle_execveat_init(const char *filename, void *envp)
-            pr_info("mark no sucompat checks for pid: '%d', exec: '%s'\n", current->pid, filename);
+            pr_debug("mark no sucompat checks for pid: '%d', exec: '%s'\n", current->pid, filename);
diff --git a/kernel/hook/setuid_hook.c b/kernel/hook/setuid_hook.c
index 43f7ce46..21b66194 100644
--- a/kernel/hook/setuid_hook.c
+++ b/kernel/hook/setuid_hook.c
@@ -87,1 +87,1 @@ int ksu_handle_setuid(uid_t new_uid, uid_t old_uid)
-        pr_info("handle_setresuid from %d to %d\n", old_uid, new_uid);
+        pr_debug("handle_setresuid from %d to %d\n", old_uid, new_uid);
